packages/components/src/combobox-control/types.ts

```diff
@@ -86,4 +86,9 @@ export type ComboboxControlProps = Pick<
 	 * If passed, the combobox input will show a placeholder string if no values are present.
 	 */
 	placeholder?: string;
+
+	/**
+	 * When loading, combobox will show a spinner
```

Comment:
For better clarity:

```suggestion
	 * Show a spinner (and hide the suggestions dropdown) while data
	 * about the matching suggestions (ie the `options` prop) is loading
```

We should also add this prop to the component's README file

Reply:
Added prop to readme in https://github.com/WordPress/gutenberg/pull/68927/commits/374b2b887626aa2b964a4e6c2b8222d2df53b956